src/test/java/org/spongepowered/mod/event/SpongeModEventManagerTest.java

```diff
@@ -75,6 +96,7 @@ public void testSpawn() throws ClassNotFoundException {
 
         eventBus.register(forgeListener);
 
+        this.eventManager.registerListeners(this.plugin, listener);
```

Comment:
I don't know if this is supposed to be there. Sponge relays some of its own events (such as `SpawnEntityEvent`) back to Forge. While there are no Sponge listeners, the event still needs to be fired for the Forge listeners. This messes with the test by adding a Sponge listener when the test should only have Forge listeners at this point.